### PR TITLE
[frontend] Publishing web component wrapper as an NPM package

### PR DESCRIPTION
**Pull request by sreenaths**
## What changes were proposed in this pull request?
- Publishing web component wrapper as an NPM package named vue3-webcomponent-wrapper.

## How was this patch tested?
Created a demo app @ https://github.com/sreenaths/vue3-webcomponent-wrapper-demo

### CODE REVIEW

File: vue3-webcomponent-wrapper/README.md [hunk 1]
```diff
@@ -0,0 +1,27 @@
+# vue3-webcomponent-wrapper
```
Comment (romainr) [inline vue3-webcomponent-wrapper/README.md:1]: Could we move this to something like https://github.com/cloudera/hue/tree/master/tools/ and not the root of the project?

Comment (sreenaths) [reply vue3-webcomponent-wrapper/README.md]: Thanks, done.

File: tools/vue3-webcomponent-wrapper/README.md [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+One main blocker preventing the official wrapper from upgrading was the lack of shadow-root CSS injection in Vue 3 build tooling. As we could live without a shadow dom in Hue this was not an issue and this port was created.
+
+## Usage
+
+    import { createApp, h } from "vue";
+    import wrapper from "vue3-webcomponent-wrapper";
+    import MyComponent from "./components/MyComponent.vue";
+
+    const webComponent = wrapper(MyComponent, createApp, h);
+    window.customElements.define("my-component", webComponent);
```
Comment (romainr) [inline tools/vue3-webcomponent-wrapper/README.md:16]: FYI: if I am someone who would be interested in the plugin, usually also having a section `Before` the plugin, aka with the official wrapper, and `After` it is released would make them trust the plugin even more? (i.e. they can very quickly see what code needs to be changed to use the new plugin as well as how to revert back in the future)

**Review by romainr — APPROVED** [review]
I can test it tomorrow too!

File: tools/vue3-webcomponent-wrapper/README.md (hunk 2 above)
Comment (sreenaths) [reply tools/vue3-webcomponent-wrapper/README.md:16]: As of now it's not clear how the official plugin will be used. Will add a note on the before part, aka how it was used in Vue 2.

**Pull request merged**